### ISSUE TITLE
Front page readme.md bloat

### ISSUE THREAD

**Issue by olife373** [j-e-k/pgoapi#550]
Hey the front page readme.md is really bloated we have a WIKI section i've been working hard to integrate it would be helpfull to start moving stuff over to the WIKI page. Perhaps even have some assistance in developing it further.

**Comment by infinitewarp** [comment]
Great work on fleshing out the wiki! I hadn't looked in there at all before now.

I agree the current `README.md` is a bit cumbersome. It should still have the essentials for getting up and running, but I think the verbose config details may be better suited for the wiki and/or stuck in a different `.md` file. (I'm a huge fan of keeping _all_ the documentation bundled with the repo so it's truly portable.)

**Comment by jekirl** [comment]
Agreed, the readme has gotten far too bloated, that is for sure. Thanks for fleshing out the wiki 😄

**Comment by olife373** [comment]
J-e-k how do I go about getting permission to commit to the wiki?

**Comment by jekirl** [comment]
Let me see....

**Comment by jekirl** [comment]
I don't see an option other than marking you as a contributor, do you know of a way?

**Comment by olife373** [comment]
Perhaps I need to fork the wiki then post commits.  Sorry Im new to get hub so I believe that may be the only way. However I do intend on heavily developing the wiki.

**Comment by olife373** [comment]
So after some additional research that too is not possible will continue to look into this after I'm off work.

**Comment by infinitewarp** [comment]
I'm confused… aren't you already contributing to the [wiki](https://github.com/j-e-k/poketrainer/wiki)? Or do you mean you want to change the README?

If the latter, yeah, your best route it to fork the repo and submit your changes to the README through pull requests.

**Comment by olife373** [comment]
Sorry for the confusion apparently there is no easy way to edit the wiki locally and push changes or even submit pull requests. @infinitewarp
